pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -393,5 +398,60 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 		}
 	}
 
+	kubeReserved, err := ParseResourceList(kc.KubeReserved)
+	if err != nil {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: unable to parse kubeReserved (--kube-reserved) %v, error: %w", kc.KubeReserved, err))
+	}
+	systemReserved, err := ParseResourceList(kc.SystemReserved)
+	if err != nil {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: unable to parse systemReserved (--system-reserved) %v, error: %w", kc.SystemReserved, err))
+	}
+
+	switch kc.CPUManagerPolicy {
+	case kubeletconfig.NoneCPUManagerPolicy:
+	case kubeletconfig.StaticCPUManagerPolicy:
+		if reservedCPUSet.Size() == 0 {
```

Comment:
Prefer early exit: `if reservedCPUSet.Size() != 0 { break }` for cleaner flow and fewer indents.